GPU/Common/TransformCommon.cpp

```diff
@@ -147,12 +147,13 @@ void Lighter::Light(float colorOut0[4], float colorOut1[4], const float colorIn[
 		Color4 lightDiff(lcolor[1][l], 0.0f);
 		Color4 diff = (lightDiff * *diffuse) * dot;
 
-		// Real PSP specular
-		Vec3f toViewer(0, 0, 1);
 		// Better specular
 		// Vec3f toViewer = (viewer - pos).NormalizedOr001(cpu_info.bSSE4_1);
 
 		if (doSpecular) {
+			// Real PSP specular
+			Vec3f toViewer(0, 0, 1);
```

Comment:
make static const

Reply:
fixed it here https://github.com/hrydgard/ppsspp/pull/18581/commits/ca94de8d4b1e53b7d596d31e7336ac5133a32a7c